content/en/dora_metrics/setup/deployments.md

```diff
@@ -30,12 +33,100 @@ further_reading:
 
 Deployment events are used to compute [deployment frequency](#calculating-deployment-frequency), [change lead time](#calculating-change-lead-time), and [change failure rate](#calculating-change-failure-rate). 
 
-## Selecting a deployment data source
+## Configuring a deployment data source
```

Comment:
I would keep the word "selecting" because the customer cannot configure the two data sources. They have to *select* one of them + configuring it

So we can put `Selecting` or `Selecting and configuring`